src/map/packet_system.cpp

```diff
@@ -3825,7 +3827,12 @@ void SmallPacket0x064(map_session_data_t* const PSession, CCharEntity* const PCh
         return;
     }
 
-    memcpy(&PChar->keys.tables[KeyTable].seenList, data[0x08], 0x40);
+    // memcpy(&PChar->keys.tables[KeyTable].seenList, data[0x08], 0x40);
+    // TODO: verify this works?
+    for (int i = 0; i < 0x40; i++)
+    {
+        PChar->keys.tables[KeyTable].seenList[i] = data[0x08 + i];
```

Comment:
Please double check this. No idea if this is the correct syntax, memset specifically was called out as bad to use against std::bitset by gcc.

Reply:
https://stackoverflow.com/questions/18513259/how-to-initialize-a-bitset-type-all-bits-to-1

Looks like `set` is a good option here for `bitset`

Reply:
Looking at it more, memset here was copying 64 bytes and this for loop is only setting 64 bits. So this implementation is definitely wrong